packages/components/select/src/select.ts

```diff
@@ -249,9 +253,9 @@ export class Select<T = unknown> extends ObserveAttributesMixin(FormControlMixin
     super.firstUpdated(changes);
 
     requestAnimationFrame(() => {
-      this.button.setAttribute('aria-controls', this.listbox.id);
```

Comment:
Why there is no `aria-controls`?

Reply:
If we really want to remove `aria-controls`, please remove it as well from the documentation, from the accessibility WAI ARIA description. :)

Reply:
I removed `aria-controls` because it won't work: `<sl-select-button>` is in the light DOM. `<sl-listbox>` is in the shadow DOM. And ARIA can't cross it. Even though those attributes are present on `main`, it doesn't work there. So to avoid confusion, i've removed it entirely here.